### DISCUSSION TITLE
Favorite recordings, unable to manage?

### DISCUSSION THREAD

**Discussion by abctje** [blakeblackshear/frigate#5360] [Ideas]
First of all, Frigate is a great addition to Home Assistant. It has almost everything needed for my purpose. 

I have applied some recordings with the favorite star. But after doing so it is not so easy to find them or manage them in an easy way. I didn't find this in the documentation. Did I miss this or does it just not exist.

It would be great to have some kind of option to manage (select, view, download selection, etc) the favorites. Maybe this can be added at some point in the future?

Keep up with the great work!

**Comment by NickM-27** [comment]
0.12 supports filtering to only favorited events.

**Reply by abctje** [reply]
That's great, thanx!
